src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +100,33 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files and adds metadata to overview-features page.
+     *
+     * @param propertiesFiles
+     *            property files to read
+     */
+    public void parsePropertiesFiles(List<String> propertiesFiles) {
+        if (propertiesFiles != null) {
```

Comment:
look at other array null-checker and use `isNotEmpty()` from utility class

Reply:
this ok for you?

```
if (propertiesFiles != null) {
   for (String propertyFile : propertiesFiles) {
        if (StringUtils.isNotEmpty(propertyFile)) {
           processClassificationFile(propertyFile);
        }
   }
}
```

I don't see any other array null-checker in the code.

Reply:
after a bit more searching...i think this is what you want?

```
if (isNotEmpty(propertiesFiles)) {
   for (String propertyFile : propertiesFiles) {
        if (StringUtils.isNotEmpty(propertyFile)) {
           processClassificationFile(propertyFile);
        }
   }
}
```

Reply:
https://commons.apache.org/proper/commons-collections/javadocs/api-release/org/apache/commons/collections4/CollectionUtils.html#isNotEmpty(java.util.Collection) should be already imported to the project

Reply:
yes that is what I'm using...will be in next PR